src/grasp_generator.cpp

```diff
@@ -978,16 +983,17 @@ bool GraspGenerator::generateSuctionGrasps(const Eigen::Affine3d& cuboid_top_pos
   ideal_grasp.translation() = cuboid_center_top_grasp.translation();
   setIdealGraspPose(ideal_grasp);
 
-  bool debug_first_transform = false;
-
-  if (debug_top_grasps_ && debug_first_transform)
+  if (debug_top_grasps_)
   {
-    visual_tools_->prompt("publish cuboid pose");
-    visual_tools_->publishAxis(cuboid_center_top_grasp, rviz_visual_tools::MEDIUM, "cuboid_center_top_grasp");
+    // visual_tools_->prompt("publish cuboid pose");
```

Comment:
I would remove all commented prompts if they are not needed.

Reply:
done